### PR TITLE
Contrast fix for hangup overlay 

### PR DESCRIPTION
**Pull request by iaulakh**
## Purpose

* Contrast fix for hangup overlay

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Review by waynemo — CHANGES_REQUESTED** [review]

File: AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/SetupActivity.java [hunk 1]
```diff
@@ -241,7 +241,6 @@ private void startCall() {
                 final JoinCallConfig joinCallConfig = new JoinCallConfig(
                         joinId, !audioToggleButton.isChecked(), videoToggleButton.isChecked(),
                         setupName.getText().toString(), callType);
-                finishAffinity();
```
[the remark this answers is not in the document]
Comment (iaulakh) [reply AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/SetupActivity.java]: ok

File: AzureCalling/app/src/main/res/layout-land/activity_call.xml [hunk 2]
```diff
@@ -117,7 +117,7 @@
         android:id="@+id/call_hangup_overlay"
         android:layout_width="0dp"
         android:layout_height="0dp"
-        android:background="#66000000"
+        android:background="#CC000000"
```
Comment (pavelprystinka) [inline AzureCalling/app/src/main/res/layout-land/activity_call.xml:120]: Does it look good on the night mode?
Did you consider defining this color at values/colors.xml with a night mode?

**Review by pavelprystinka — CHANGES_REQUESTED** [review]

File: AzureCalling/app/src/main/res/layout-land/activity_call.xml (hunk 2 above)
Comment (pavelprystinka) [reply AzureCalling/app/src/main/res/layout-land/activity_call.xml:120]: Great, thanks Inderpal

**Review by pavelprystinka — APPROVED** [review]

**Review by waynemo — APPROVED** [review]

**Comment by waynemo** [comment]
When squashing, remove some of the intermediate commit messages from the description.

**Pull request merged**